Project/src/MakeCall/VideoSendStats.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+
+export default class VideoSendStats extends React.Component {
```

Comment:
nit: could be functional component. Both functional and class component has pros and cons. Looks like functional component is preferable and the trends.